ui/raidboss/data/05-shb/eureka/bozjan_southern_front.js

```diff
@@ -1113,5 +1142,84 @@ export default {
         'Winds\' Peak': '超级烈风波',
       },
     },
+    {
+      'locale': 'ko',
+      'missingTranslations': true,
+      'replaceSync': {
+        '4Th Legion Helldiver': 'IV군단 헬다이버',
+        'Adrammelech': '아드람멜렉',
+        'Bladesmeet': '검들의 대광장',
+        'Brionac': '브류나크',
+        'Dawon': '다우언',
+        'Eaglesight': '독수리 광장',
+        'Lightsphere': '빛 구체',
+        'Lyon The Beast King': '마수왕 라이언',
+        'Majesty\'s Auspice': '단상',
+        'Shadowsphere': '그림자 구체',
+        'The airship landing': '골드 소서 비공정 승강장',
+        'The grand gates': '성문',
+        'Verdant Plume': '진녹색 날개',
+      },
+      'replaceText': {
```

Comment:
```suggestion
      'replaceText': {
        '--Lyon Passage--': '--라이언 통과--',
```
I don't exactly know about this, I translated this guessing this means that Lyon pass through something.